README.md

```diff
@@ -152,28 +93,85 @@ When you first clone a repository with submodules, you’ll need to initialize a
-
-    remote: Permission to crdschurch/crds-net-shared.git denied to [USERNAME].
-    fatal: unable to access 'https://github.com/crdschurch/crds-net-shared.git/': The requested URL returned error: 403
-
+```bash
+remote: Permission to crdschurch/crds-net-shared.git denied to [USERNAME].
+fatal: unable to access 'https://github.com/crdschurch/crds-net-shared.git/': The requested URL returned error: 403
+```
 It's likely that where you see `[USERNAME]` does not match your GitHub username. If on a Mac, this means OS X Keychain has stored a value for your GitHub username and password that is being used wherever you use HTTPS (rather than SSH) to push and pull code from GitHub.
 
 To remedy this, you first have to delete the stored value:
-
-    $ git credential-osxkeychain erase
-     host=github.com
-     protocol=https
-     <press return>
-
+```bash
+$ git credential-osxkeychain erase
+  host=github.com
+  protocol=https
+  <press return>
+```
 This behavior is odd. After hitting `Enter` after the first line, nothing happens -- you are directed to a blank line in the terminal. That's where you type in `host=github.com` and hit `Enter` again. Then followed by the next line. The third empty line you leave blank and hit `Enter` for a fourth time. Then your credentials should be stored.
 
 If you try to push again, you should be prompted for a new username and password and those will be stored in your keychain for future use.
 
-## License
+## Advanced Setup
+### Building Assets
+`crds-net` uses `jekyll-asset-pipeline` to create Javascript/CSS assets. If you see that assets are not showing up make sure to check that [the pipeline is installed correctly](https://github.com/crdschurch/jekyll-asset-pipeline). If all else fails, try `rm -rf _site` and then start Jekyll back up.
 
-This project is licensed under the [3-Clause BSD License](https://opensource.org/licenses/BSD-3-Clause).
+### OS Specific Issues
+#### Mac
+To get started with this project, first make sure you have Ruby installed. This
+project requires the use of Ruby version >= 2.5.1. If you are on a Mac, Ruby should
+already be installed but you may need to upgrade your version. rbenv is a great tool
+to manage your Ruby versions. You can read about that [here](https://github.com/rbenv/rbenv#installation).
+Once you have Ruby version >= 2.5.1 installed, make sure you have Ruby bundler
+installed.
+`gem install bundler`
+After installing bundler, you will be able to install the project's dependencies that
+are defined in the `Gemfile`. You can do that by running
+`bundle install`
+Once the project dependencies are installed, you can now kick off a build of the
+project:
+`bundle exec jekyll serve`
+This will build the site and start up a preview server that allows you to view the
+work that you have done.
+
+####  Windows
+First, make sure you have Ruby v2.5.1 installed on your machine. You can find Windows installers (with devkit) [here](https://rubyinstaller.org/downloads/).
```

Comment:
The Windows specific section now only needs to include these two subsections:
Installing Ruby: "Install Ruby v2.5.1 (or greater) using the Windows installers (with devkit) [here](https://rubyinstaller.org/downloads/).
Serving the site: "1. Run `bundle exec jekyll build`
2. Find the asset hash value in the /temp/.asset_hash file and assign it to the 'ASSET_HASH' environment variable.
3. Run `npm run build`
4. Run `bundle exec jekyll serve --skip-initial-build`
**Some background - `bundle exec jekyll serve` serves the page but errors when loading anything from the /assets folder. This workaround makes it accessible.